packaging_ignore.yml

```diff
@@ -15,6 +15,7 @@ base:
   - ".* unused substitution variable \\$\\{misc:Depends\\}.*"
   - ".*: substitution variable \\$\\{postgresql:Depends\\} unused, but is defined.*"
   - ".* unused substitution variable \\$\\{postgresql:Depends\\}.*"
+  - ".*: WARNING: unrecognized options: --with-security-flags"
```

Comment:
I do not think we should ever ignore this line.

Reply:
You're right removed it